eZ/Publish/Core/REST/Server/Controller/Content.php

```diff
@@ -736,4 +702,66 @@ protected function forward($controller)
 
         return $this->container->get('http_kernel')->handle($subRequest, HttpKernelInterface::SUB_REQUEST);
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return mixed
+     */
+    protected function parseCreateContentRequest(Request $request)
+    {
+        return $this->inputDispatcher->parse(
+            new Message(
+                array('Content-Type' => $request->headers->get('Content-Type'), 'Url' => $request->getPathInfo()),
+                $request->getContent()
+            )
+        );
+    }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param \eZ\Publish\Core\REST\Server\Values\RestContentCreateStruct $contentCreate
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     *
+     * @return \eZ\Publish\Core\REST\Server\Values\CreatedContent
+     */
+    protected function doCreateContent(Request $request, RestContentCreateStruct $contentCreate)
+    {
+        try {
+            $content = $this->repository->getContentService()->createContent(
+                $contentCreate->contentCreateStruct,
+                array($contentCreate->locationCreateStruct)
+            );
+        } catch (ContentValidationException $e) {
+            throw new BadRequestException($e->getMessage());
+        } catch (ContentFieldValidationException $e) {
+            throw new RESTContentFieldValidationException($e);
+        }
+
+        $contentValue = null;
+        $contentType = null;
+        $relations = null;
```

Comment:
This is some light Friday OCD kicking in, but this could also be written as:
` $contentValue = $contentType = $relations = null;`

or 
```
$relations = null;
$contentType = null;
$contentValue = null;
```

;)